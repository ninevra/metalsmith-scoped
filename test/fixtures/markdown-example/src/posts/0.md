---
description: a post
---
Some **markdown**.
