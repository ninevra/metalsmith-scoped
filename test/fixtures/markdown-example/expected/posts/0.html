<p>Some <strong>markdown</strong>.</p>
